### PR TITLE
Add new entry about prototype model in SDLC

### LINKED ISSUE #1806: [Term Entry] General SDLC: Prototype
### Reason/inspiration (optional)

The [Software Development Lifecycle](https://www.codecademy.com/resources/docs/general/software-development-life-cycle) Model to cover:

- Agile: Iterative development and continuous integration. The scrum methodology is a common implementation of the agile model in which teams work in sprints. Sprints are time-constrained periods where a team works to complete a certain goal.

Please write a new entry and expand on the above introductory paragraph under:  content/general/concepts/software-development-life-cycle/terms/ about the Prototype model.

To get started with the PR and entry file:

Under content/general/concepts/software-development-life-cycle/terms/, create a new folder called `prototype`. Then create a new file called `prototype.md` under content/general/concepts/software-development-life-cycle/terms/`prototype`

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by onekir**
### Description

Closes #1806 
Add new entry about prototype model in SDLC

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Review by SSwiniarski — APPROVED** [review]
@onekir looks good. I'm sending it on for a second review.

**Comment by onekir** [comment]
> @onekir looks good. I'm sending it on for a second review.

@SSwiniarski Hi! Do I need to do something or just wait for the next review?

**Comment by SSwiniarski** [comment]
> > @onekir looks good. I'm sending it on for a second review.
> 
> @SSwiniarski Hi! Do I need to do something or just wait for the next review?

@onekir Just wait for the second review.

File: content/general/concepts/software-development-life-cycle/terms/prototype/prototype.md
Comment (yangc95) [inline content/general/concepts/software-development-life-cycle/terms/prototype/prototype.md:44]:
```suggestion
1.  Requirements gathering: At this stage, the project team communicates with customers, conducts research, and collects all the requirements for the future product.
2.  Design: At this stage, the structure and design of the product are determined. For example, for a website, this would include the number of pages, the purpose of each page, and the placement of key elements.
3.  Prototyping: Creating a prototype using the design that was developed in the previous stage.
4.  Prototype testing and feedback collection: The developed prototype is tested to identify errors and areas for improvement.
5.  Prototype refinement: The feedback obtained from testing is used to improve the prototype and meet the requirements of the product.
6.  Development: After the prototype has been refined and meets customer requirements, the team moves on to the final product development.
```

**Review by yangc95 — COMMENTED** [review]
@onekir great entry, very well-written.

I left a few comments and suggestions so if you have any questions do let me know.

Thanks for your patience throughout this process 🙂

**Comment by caupolicandiaz** [comment]
@onekir There hasn't been an update on this entry in many days, are you still available to complete this entry?

**Review by onekir — COMMENTED** [review]
@yangc95 Thank you for your comments and suggestions. I have reviewed them and I agree with all the proposed changes.

**Comment by onekir** [comment]
> @onekir There hasn't been an update on this entry in many days, are you still available to complete this entry?

Yes, I am still available to complete this entry. I have agreed with all the proposed changes.

**Comment by SSwiniarski** [comment]
@onekir if you agree with the proposed changes, you still need to edit them into this PR.

File: content/general/concepts/software-development-life-cycle/terms/prototype/prototype.md
Comment (yangc95) [reply content/general/concepts/software-development-life-cycle/terms/prototype/prototype.md:20]: @onekir

**Review by yangc95 — APPROVED** [review]

**Comment by yangc95** [comment]
🙌 @onekir here's the live link: https://www.codecademy.com/resources/docs/general/software-development-life-cycle/prototype

Screenshot 2023-06-05 at 7 25 21 PM

It looks as if you're anonymous. If you want to be credited follow the instructions below to add your GitHub username to your Codecademy profile. If not, there is nothing more you need to do.
To add your GitHub username to your Codecademy profile:
1.) Add your GitHub Username to your Codecademy Profile Details, you can get there on your desktop by pressing on your profile picture in the upper right-hand corner and selecting "Profile" in the drop-down box:
githubusername

Make sure to set your visibility to “everyone”:
Screenshot 2023-04-10 at 4 07 30 PM


2.) Then log out and log back into Codecademy using your GitHub login:
githublogin

If the steps above do not work, it may be due to an email or local configuration issue.
In addition, we may reach out in the future if any adjustments to these entries are required.


cc: @SSwiniarski

**Pull request merged**
